pkg/simulation/simulation.go

```diff
@@ -53,6 +54,14 @@ type simulation struct {
 	actionTargets map[key.TargetID]bool
 }
 
+func RunWithLog(cfg *model.SimConfig, eval *eval.Eval, seed int64) (*model.IterationResult, error) {
```

Comment:
imo `RunWithLog` should take in a `Logger` and perform the singleton assignment rather than relying on the caller to do so